apps/api/src/config/env.config.ts

```diff
@@ -5,6 +5,10 @@ import { mnemonicValidate } from '@polkadot/util-crypto';
 export const configModuleOptions: ConfigModuleOptions = {
   isGlobal: true,
   validationSchema: Joi.object({
+    IPFS_ENDPOINT: Joi.string().uri().default('https://ipfs.infura.io:5001'),
```

Comment:
do we want to have defaults or maybe we should set everything as required just like REDIS_URL. I think keeping it consistent across all configs would be ideal.

Reply:
Done!